packages/eslint-config-finn-prettier/prettier.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+module.exports = {
+    extends: 'prettier',
+    plugins: [
+        'prettier',
+    ],
+
+    rules: {
+        prettier: ['error', { printWidth: 120, tabWidth: 4, singleQuote: true, trailingComma: 'es5' }],
```

Comment:
I'd be fine with the 80 width default. Prettier has a tendency to inline too much of chained expressions